Automattic/team51-cli
# Update create-repository command

Closes #30 

This PR is just to update the `create-repository` command. I removed Travis files, added GH Action files.

To test:
1. Switch to this branch locally
2. Run `create-repository`
3. New repo should run GH Action for PHPCS checks, rather than Travis
